service/inference.py

```diff
@@ -44,6 +49,12 @@
     expose_headers=["x-sample-id", "x-request-id", "x-metrics"]
 )
 
+global whisper_model
+whisper_model = whisper.load_model("small")
+
+OpenAI.api_key = os.getenv("OPENAI_API_KEY")
```

Comment:
An easier way is to store the environment variables in a .env file and use load_dotenv() to make all the environment variables available to the program.

Ref: https://pypi.org/project/python-dotenv/